morse_app/settings.py

```diff
@@ -39,8 +39,39 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'django.contrib.sites',
+
+    # 3rd party
+    'rest_framework',
+    'rest_framework.authtoken',
+    'dj_rest_auth',
+    'dj_rest_auth.registration',
+    'allauth',
+    'allauth.account',
+    'allauth.socialaccount',
+
+    # local
+    'users',
 ]
 
+# django-allauth config
+SITE_ID = 1
```

Comment:
What is this variable for?

Reply:
It's for Django-allauth to recognize the site's url